api.md

```diff
@@ -116,6 +124,7 @@ Methods:
 
 Response Types:
 
+- <a href="https://pkg.go.dev/github.com/Finch-API/finch-api-go">finchgo</a>.<a href="https://pkg.go.dev/github.com/Finch-API/finch-api-go#BenefitSupportType">BenefitSupportType</a>
```

Comment:
Let's scope down changes for this release to only include: 

• (new endpoint) /forward
• (new response field) source_id on /employment

---

So let's exclude this.